mne/utils/docs.py

```diff
@@ -2453,7 +2453,7 @@
 
 # BEM
 docdict['on_defects'] = """
-on_defects : str
+on_defects : 'raise' | 'warn'
```

Comment:
This should use the `_on_missing` docstring base and functionality, can you open a PR to fix this? ('ignore'` is missing here, and the code is not DRY.)

Reply:
I didn't know `ignore` was supported here, ugh. Yes I can open a followup PR!

Reply:
It should be as long as `on_defects` uses the `_on_missing` machinery. If it doesn't do that, it should